interface/resources/qml/hifi/Pal.qml

```diff
@@ -620,7 +623,39 @@ Rectangle {
                     HiFiGlyphs {
                         text: (styleData.role === "kick") ? hifi.glyphs.error : hifi.glyphs.muted;
                         // Size
-                        size: parent.height*1.3;
+                        size: parent.height * 1.3;
+                        // Anchors
+                        anchors.fill: parent;
+                        // Style
+                        horizontalAlignment: Text.AlignHCenter;
+                        color: enabled ? hifi.buttons.textColor[actionButton.color]
+                            : hifi.buttons.disabledTextColor[actionButton.colorScheme];
+                    }
+                }
+                
+                HifiControlsUit.Button {
+                    id: hardBanButton;
+                    color: 2; // Red
+                    visible: isBan;
+                    enabled: !nameCard.isReplicated;
+                    anchors.verticalCenter: itemCell.verticalCenter;
+                    anchors.left: parent.left;
+                    anchors.leftMargin: actionButton.width + 14;
+                    width: 32;
+                    height: 32;
+                    onClicked: {
+                        Users[styleData.role](model.sessionId, 7); // Ban params add up to 7 from Users.BAN_BY_USERNAME | Users.BAN_BY_FINGERPRINT | Users.BAN_BY_IP
```

Comment:
can you not just actually or them here?

Reply:
Uwotm8?

Reply:
like, instead of 7, we should be able to do Users.BAN_BY_USERNAME | Users.BAN_BY_FINGERPRINT | Users.BAN_BY_IP

Reply:
I made the update but will have to test this later with someone. I can't ban myself, heh.